input/src/main/scala/perceptor/invalidTO_DO.scala

```diff
@@ -0,0 +1,21 @@
+// scalafix:off
+/*
+rule = DisableSyntax
+
+DisableSyntax.regex = [
+  {
+    id = "tdo"
+    pattern = "(TODO|FIXME)"
+    message = "Must be in format <KEYWORD>(<jiraurl)"
+  }
+]
+ */
+// scalafix:on
+package perceptor
+
+class invalidTO_DO {
+
+  // TODO assert: DisableSyntax.tdo
```

Comment:
Appears to fail because the TODO is part of the comment itself and appears 'unexpected' rather than unreported